pkgs/development/python-modules/numpy/default.nix

```diff
@@ -56,6 +56,13 @@ in buildPythonPackage rec {
     ./numpy-distutils-C++.patch
   ];
 
+  postPatch = ''
+    # fails with multiple errors:
```

Comment:
```suggestion
    # fails with multiple errors because we are not using the pinned setuptools version
    # see https://github.com/numpy/numpy/blob/v1.25.0/pyproject.toml#L7
```